.github/workflows/ci-scripts-build.yml

```diff
@@ -68,17 +70,26 @@ jobs:
             extra: "CMD_CXXFLAGS=-std=c++11"
             name: "Ub-20 clang-10 C++11"
 
+          - os: ubuntu-20.04
+            cmp: gcc
+            configuration: default
+            rtems: "5"
+            rtems_target: RTEMS-pc686-qemu
```

Comment:
Please add this line to name this particular matrix entry:
```yaml
            name: "Ub-20 gcc-9 + RT-5.1"
```

Reply:
added